web/client/components/misc/enhancers/security/accessRuleParser.jsx

```diff
@@ -0,0 +1,92 @@
+/*
```

Comment:
should we move this in components/security/enhancers?

Reply:
security folder is now dedicated to user, group, and permission editor. This is a different thing, so I'm afraid placing it there may be misleading